### ISSUE TITLE
Accessibility: WCAG2AA.Principle1.Guideline1_3.1_3_1.F68

### ISSUE THREAD

**Issue by funkysi1701** [funkysi1701/funkysi1701.github.io#2826]
## Accessibility Issue Found by Pa11y

**WCAG Code**: `WCAG2AA.Principle1.Guideline1_3.1_3_1.F68`
**Total Occurrences**: 1
**URL**: https://blog-dev.funkysi1701.com
**Build**: 10.1.1.2333

---

### Example 1

**Message**: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
**Selector**: `#modeSwitcher`

**Context:**
```html
<input class="form-check-input" type="checkbox" id="modeSwitcher">
```

---
_Detected by [Pa11y](https://pa11y.org/) accessibility checker in Azure Pipeline_
